tests/fixtures/xfail/default-role-separated-by-comas.rst

```diff
@@ -0,0 +1 @@
+This is not detected: `foo`, `bar`.
```

Comment:
The filename is missing an `m`.

Reply:
Not the first time I misstype comma... I'll have to learn from my errors...